tox.ini

```diff
@@ -51,7 +51,7 @@ description = run linter (rstcheck) to ensure there aren't errors on our docs
 deps = -r{toxinidir}/requirements/docs-lint.txt
 changedir = {toxinidir}/docs
 commands =
-    rstcheck -r .
+    rstcheck -r --ignore-substitutions org_brand,com_brand .
```

Comment:
These should be in https://github.com/readthedocs/readthedocs.org/blob/master/docs/.rstcheck.cfg